Stellar-ledger-entries.x

```diff
@@ -508,8 +508,29 @@ struct ContractDataEntry {
     SCVal val;
 };
 
+struct ContractCodeCostInputs {
+    uint32 nFunctions;
+    uint32 nGlobals;
+    uint32 nTableEntries;
+    uint32 nTypes;
+    uint32 nDataSegments;
+    uint32 nElemSegments;
+    uint32 nImports;
+    uint32 nExports;
+};
+
 struct ContractCodeEntry {
-    ExtensionPoint ext;
+    union switch (int v)
+    {
+        case 0:
+            void;
+        case 1:
+            struct
+            {
+                ExtensionPoint ext;
```

Comment:
Should we put (or add an additional) the `ext` inside the `ContractCodeCostInputs`, since that is more likely need to be extended with more cost factors.

Reply:
Fixed.

Reply:
Thanks, feel free to resolve